klarna-on-demand/src/main/java/com/klarna/ondemand/HelperMethods.java

```diff
@@ -5,4 +5,12 @@
     static Boolean isBlank(String str) {
         return str == null || str.isEmpty();
     }
+
+    static String hexStringFromColor(Integer color) {
```

Comment:
I just saw this guy now. This is starting to smell a little non-trivial. Just be sure people end up being able to easily understand how to change the color of various things. Documentation is your friend and all that.